src/main/java/com/schematica/world/storage/Schematic.java

```diff
@@ -20,17 +20,17 @@
 
 public class Schematic implements ISchematic
 {
-    private static final ItemStack                              DEFAULT_ICON   = new ItemStack(Blocks.grass);
+    private static final ItemStack                              DEFAULT_ICON   = new ItemStack(Blocks.GRASS);
     private static final FMLControlledNamespacedRegistry<Block> BLOCK_REGISTRY = GameData.getBlockRegistry();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "getBlockRegistry"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)